modules/ROOT/pages/logging-in-mule.adoc

```diff
@@ -0,0 +1,284 @@
+= Configuring Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, studio, logger, logs, log, notifications, errors, debug
+
+Mule automatically logs multiple messages and specific elements in your application flows to help you debug and keep track of events. You can also include the xref:logger-component-reference.adoc[Logger component] anywhere in a flow and set it up to output any message you want.
+
+You can create a configuration file to define:
+
+ * What kinds of messages to log
+ * How to log the messages (asynchronously or synchronously)
+ * Where to log the messages, such as to the console or disk, or to an endpoint or database
+
+Mule uses `slf4j`, which is a logging facade that discovers and uses a logging strategy from the classpath, such as Apache Log4j 2 or the JDK Logger. By default, Mule includes Log4j 2, which is configured with a file called `log4j2.xml`.
+
+The Mule server has a `log4j2.xml` file in its `conf` directory, which you can customize when running the server in standalone mode. This configuration file is relevant for data output to the `mule_ee.log` file.
+
+If you remove the `log4j2.xml` file from your application's classpath, Mule Runtime applies a default logging configuration to the application and outputs the log to `MULE_HOME/logs/<application-name>.log`.
```

Comment:
We should first mention that each application that you build on Studio, for example, comes with its own log4j2.xml, which is the file this sentence is referring to. Otherwise, right now it gets confused with the Mule server own log4j2.xml mentioned in the previous sentence.